src/main.js

```diff
@@ -15,7 +15,7 @@ function createWindow () {
   // and load the index.html of the app.
   mainWindow.loadURL(`file://${__dirname}/index.html`)
 
-  // mainWindow.webContents.openDevTools()
+  mainWindow.webContents.openDevTools()
```

Comment:
Ah yeah, I was doing that then got lazy. I'll comment this back out, didn't intend to check it in!